miner/worker.go

```diff
@@ -1032,6 +1032,9 @@ func (w *worker) prepareWork(genParams *generateParams) (*environment, error) {
 	}
 	if genParams.gasLimit != nil { // override gas limit if specified
 		header.GasLimit = *genParams.gasLimit
+	} else if w.chain.Config().Optimism != nil && w.config.GasCeil != 0 {
+		// configure the gas limit of pending blocks with the miner gas limit config when using optimism
+		header.GasLimit = w.config.GasCeil
 	}
```

Comment:
Does the pending block go on to be sealed or is it only used for fielding `pending` RPC queries and a new block is built for the next block? With a 2s blocktime, the concept is `pending` is not as useful

Reply:
Geth is always building & sealing a `pending` block, regardless of requests.

Reply:
You're right about the block time, but we don't want to break `pending` either. And with the current recommit time for block building the `pending` block is kind of silly.

Reply:
I see your test in this PR https://github.com/ethereum-optimism/optimism/pull/4047 and it answers my question. Good work!